src/DateComparison.php

```diff
@@ -16,4 +17,11 @@ public static function onSameDay(DateTimeInterface $date1, DateTimeInterface  $d
 
         return $date1->format('Y-m-d') === $date2->format('Y-m-d');
     }
+
+    public static function inTheFuture(DateTimeInterface $date): bool
```

Comment:
I feel like we're implementing `Carbon` here 😅

Reply:
True. Can we do it like this for now, and depending on my question in your PR about decoupling from `search-v3` and just doing the deserialization ourselves we can use `Carbon` or `Chronos` instead for the dates?

Reply:
An interesting extra remark about this: we sometimes do `->setTimezone()` which is technically not a part of `DateTimeInterface` 🙃